src/negotiation/order.ts

```diff
@@ -58,6 +79,14 @@ export function isNative({ asset, ledger }: OrderAsset): boolean {
 const BITCOIN_DECIMALS = 8;
 const ETHER_DECIMALS = 18;
 
+/**
+ * Convert from nominal amount to underlying base unit of an asset, e.g. convert Ether to Wei.
+ *
+ * @param asset Name of the asset as string (e.g. "ether".
+ * @param nominalAmount The nominal amount (e.g. ether amount).
+ * @param token Optional parameter for converting ERC20 tokens according to defined {@link Token.decimals}.
+ * @returns The base unit amount of the asset (e.g. amount in wei).
```

Comment:
```suggestion
 * @returns The smallest unit amount of the asset (e.g. amount in wei).
```

Reply:
Not even sure my proposal is correct either as lightning deals with millisats

Reply:
maybe `native` unit? not sure, would keep base for now.